### PR TITLE
upgrade golang.org/x/crypto dep to avoid CVE

### PR DESCRIPTION
**Pull request by sipsma**
CVE doesn't seem especially relevant to us, but need this to avoid CVE scan failures.

Example failure: https://github.com/dagger/dagger/actions/runs/12287348922/job/34289228329?pr=9171#step:3:3362

### CODE REVIEW

File: sdk/typescript/runtime/go.mod
Comment (jedevc) [inline sdk/typescript/runtime/go.mod:1]: :thinking: hm, this is a dagger module, this probably needs a `dagger develop` first, these dependencies shouldn't be wiped.

File: sdk/php/runtime/go.mod
Comment (jedevc) [inline sdk/php/runtime/go.mod:1]: Likewise with the php module here.

File: sdk/elixir/runtime/go.mod
Comment (jedevc) [inline sdk/elixir/runtime/go.mod:1]: And elixir.

**Review by jedevc — APPROVED** [review]

File: sdk/typescript/runtime/go.mod
Comment (sipsma) [reply sdk/typescript/runtime/go.mod:1]: Oops yeah I was overzealous with a `find` somewhere, re-ran `dagger develop` on all the sdk runtimes

**Review by jedevc — APPROVED** [review]
Ah, looks like some code was still using ye-olde unscoped enum values (breaking linting).

Otherwise, LGTM, good to merge when we're green :tada:

**Pull request merged**